<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin_Dialog</class>
 <widget class="QDialog" name="Admin_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>756</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Application de gestion de réservation du vol</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>691</width>
     <height>411</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>High Tower Text</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="icon">
     <iconset resource="MyResource.qrc">
      <normaloff>:/new/prefix1/Data/Pictures/welcomeIcone.jpg</normaloff>:/new/prefix1/Data/Pictures/welcomeIcone.jpg</iconset>
    </attribute>
    <attribute name="title">
     <string>Welcome</string>
    </attribute>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_bienvenue_admin">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="icon">
     <iconset>
      <normalon>:/new/prefix1/Data/Pictures/mangeUser.png</normalon>
     </iconset>
    </attribute>
    <attribute name="title">
     <string>Manage users</string>
    </attribute>
    <widget class="QLabel" name="label_login">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>120</y>
       <width>201</width>
       <height>201</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <stylestrategy>PreferAntialias</stylestrategy>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>20</y>
       <width>231</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>High Tower Text</family>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Click on &quot;Read&quot; to open the file,
or click on &quot;Write&quot; to modify it.
</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>391</width>
       <height>311</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPlainTextEdit" name="plainTextEdit"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="font">
           <font>
            <family>High Tower Text</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Write</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="font">
           <font>
            <family>High Tower Text</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Read</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="bguser">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>701</width>
       <height>461</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <zorder>bguser</zorder>
    <zorder>label_login</zorder>
    <zorder>label_2</zorder>
    <zorder>layoutWidget</zorder>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="icon">
     <iconset>
      <normalon>:/new/prefix1/Data/Pictures/mangeUser.png</normalon>
     </iconset>
    </attribute>
    <attribute name="title">
     <string>Manage reservations</string>
    </attribute>
    <widget class="QLabel" name="label_dbState">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>360</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>255</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>255</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>120</red>
           <green>120</green>
           <blue>120</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Light</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="bguser_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>701</width>
       <height>461</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>140</y>
       <width>205</width>
       <height>81</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="gestionVol">
        <property name="text">
         <string>Flight management</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="gestionVoyageur">
        <property name="text">
         <string>Traveler Management</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <zorder>bguser_2</zorder>
    <zorder>label_dbState</zorder>
    <zorder>layoutWidget_2</zorder>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="icon">
     <iconset>
      <normalon>:/new/prefix1/Data/Pictures/aboutIcone.png</normalon>
     </iconset>
    </attribute>
    <attribute name="title">
     <string>About</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>480</width>
       <height>104</height>
      </rect>
     </property>
     <property name="text">
      <string>Sujet 7 : Application de gestion de réservation du vol
L’objectif est de développer une application graphique permettant de gérer les réservations des vols
au niveau d’une agence de voyage. L’utilisateur de cette application peut :
 Ajouter un vol (numéro vol, date et heure départ, date et heure d’arrivée, Aéroport départ,
aéroport d’arrivée, numéro avion)
 Rechercher et consulter des vols existant par date départ, destination ..., etc.
 Ajouter un voyageur et lui réserver un vol
 Il peut aussi modifier ou annuler un vol.</string>
     </property>
    </widget>
    <widget class="QLabel" name="bguser_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>701</width>
       <height>461</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <zorder>bguser_3</zorder>
    <zorder>label</zorder>
   </widget>
  </widget>
  <widget class="QLabel" name="label_background2">
   <property name="geometry">
    <rect>
     <x>-6</x>
     <y>0</y>
     <width>971</width>
     <height>501</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>440</y>
     <width>701</width>
     <height>43</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_time">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>113</red>
          <green>113</green>
          <blue>113</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>113</red>
          <green>113</green>
          <blue>113</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_date">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>107</red>
          <green>107</green>
          <blue>107</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>107</red>
          <green>107</green>
          <blue>107</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>741</width>
     <height>481</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <zorder>label_background2</zorder>
  <zorder>frame</zorder>
  <zorder>tabWidget</zorder>
  <zorder>groupBox</zorder>
 </widget>
 <resources>
  <include location="MyResource.qrc"/>
 </resources>
 <connections/>
</ui>
